jameda_interface_config.js

```diff
@@ -279,4 +279,39 @@ var interfaceConfig = {
     */
 };
 
+/**
+ * Checks if the browser is mobile.
+ * @returns {?boolean}
+ */
+function isMobileBrowser() {
+    let check = false;
+
+    (function(a) {
```

Comment:
You don't have to use IIFE. Maybe try to sign to variable result of matched user agent.
e.g.
const regex = ...

function isMobileBrowser() {
    return regex.test(userAgent);
}

Reply:
There is no need to create global variables which are used only once. Therefore IIFE in this case is useful due to we only need this piece of code to be executed once.

Reply:
Ok, then stay with your solution :)